### DISCUSSION TITLE
Can I use it in CC Tweaked?

### DISCUSSION THREAD

**Discussion by SilverSCode** [MCJack123/craftos2#278] [Q&A]
I'm not sure if I can install it in CC tweaked mod? Is it a standalone application?

**Comment by MCJack123** [comment]
It's a standalone application for now, but installing CC:T gives you most of the same features in-game. Work has started on a mod to integrate CraftOS-PC in-game, but this is still in very early stages, and no estimate on release is available.

![CC2 early demo](https://media.discordapp.net/attachments/477911902152949771/933743908440141834/unknown.png)
